Bucketeer2/Sources/Internal/Database/SQLite/SQLite.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import SQLite3
+
+final class SQLite {
```

Comment:
This is a SQLite client created not to depend on a library.
It is in reference to [SQLite.swift](https://github.com/stephencelis/SQLite.swift)